packages/tools/devtools/devtools-view/src/components/ContainerSummaryView.tsx

```diff
@@ -233,6 +238,13 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 				}
 				return false;
 			},
+			[ContainerDevtoolsFeatures.MessageType]: async (untypedMessage) => {
+				const message = untypedMessage as ContainerDevtoolsFeatures.Message;
+				if (message.data.containerKey === containerKey) {
+					return true;
+				}
+				return false;
+			},
```

Comment:
Do we need this? We're getting the features via `useContainerFeaturesContext`. Same question for the message we're posting on line 268.